config/prow/cluster/monitoring/prow_prometheus.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         resources:
           requests:
             storage: 100Gi
```

Comment:
Do we have any idea how much of this we're using?

Reply:
I don't have access to check in on the PV itself.  Going off of prometheus' own metrics, storage consumed for its data has grown as follows:

![Screen Shot 2020-09-22 at 4 57 07 PM](https://user-images.githubusercontent.com/49258/93936780-af361b80-fcf4-11ea-9651-48c8aaf6569e.png)

We're at 40gb now after a little under 90d.  Doesn't look like we'll hit our 100Gi limit in 180d.  We'll need to resize if we want retention any further than that.

WDYT @cjwagner

Reply:
Sounds safe to retain 180d then.
If resizing necessitates restarting with no data then we should resize sooner rather than later, but presumably we can instead migrate the data to new PVs or resize the PVs in place? I'm not sure what the process would be for a PV managed by the prometheus operator.

Reply:
It sounds like it's doable manually, and there may be a stalled upstream KEP preventing it from happening automatically (ref: https://github.com/prometheus-operator/prometheus-operator/issues/2357)

Reply:
Opened https://github.com/kubernetes/test-infra/issues/19325 for followup